app/graphql/types/billable_metrics/object.rb

```diff
@@ -25,6 +27,14 @@ def group
       def flat_groups
         object.selectable_groups
       end
+
+      def active_subscriptions_count
+        object.plans.joins(:subscriptions).merge(Subscription.active).count
+      end
+
+      def draft_invoices_count
+        object.plans.joins(subscriptions: [:invoices]).merge(Invoice.draft).count
```

Comment:
Since we are going through subscription to get invoices don't we have a risk to get duplication in the count if an invoice is linked to multiple subscription for the same plan?

Reply:
Oh seems like a nice catch! 👏  Looking into this!